### ISSUE TITLE
모델 학습,평가 진행상황

### ISSUE THREAD

**Issue by 17011775** [SEEUNL/Recycle#18]
VGGNet ( training set : validation set = 10 : 1, Data Augmentation Off, )
batch_size = 3, step_per_epoch = 1500, epochs = 50 : 학습 acc 0.97, 평가 acc 0.53
batch_size = 3, step_per_epoch = 1500, epochs = 35 : 학습 acc 0.97, 평가 acc 0.33
batch_size = 3, step_per_epoch = 1500, epochs = 30 : 학습 acc 0.96, 평가 acc 0.66
batch_size = 3, step_per_epoch = 1500, epochs = 25 : 학습 acc 0.95, 평가 acc 0.33

batch_size = 20, step_per_epoch = 100, epochs = 15 : 학습 acc 0.80, 평가 acc 0.61
batch_size = 20, step_per_epoch = 100 epochs = 20 : 학습 acc 0.88, 평가 acc 0.47

ResNet (training set : validation set = 10 : 1, Data Augmentation Off)
batch_size = 20, step_per_epoch = 100, epochs = 50 : 학습 acc 0.98, 평가 acc 0.5

![KakaoTalk_20190605_233916514](https://user-images.githubusercontent.com/48524429/58965396-84684100-87eb-11e9-8b05-51fb96e59543.jpg)
validation accuracy, loss값 변동이 심하고 진전이 잘 안됨-> overfitting에 빠짐  
다음번엔 데이터셋 분배 비율을 조정, 데이터 추가 수집, 데이터 어그멘테이션을 적용시킬 예정
